.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  ignorePatterns: ["/node_modules/**", "/dist/**", "/util/**", "index.d.ts", "compile.ts"],
+  extends: [
```

Comment:
We can just extend @fig/eslint-config-fig. It already extends these 4 things and configures them in a consistent way across repos